internal/charmstore/elasticsearch.go

```diff
@@ -49,313 +49,285 @@ const esIndexJSON = `
 
 const esMappingJSON = `
 {
-  "entity" : {
-    "dynamic" : "false",
-    "properties" : {
-      "URL" : {
-        "type" : "multi_field",
-        "fields" : {
-          "URL" : {
-            "type" : "string",
-            "index" : "not_analyzed",
-            "omit_norms" : true,
-            "index_options" : "docs"
-          },
-          "ngrams" : {
-            "type" : "string",
-            "analyzer" : "n3_20grams",
-            "include_in_all" : false
-          }
-        }
-      },
-      "PromulgatedURL" : {
-        "type" : "string",
+  "entity": {
+    "dynamic": "false",
+    "properties": {
+      "URL": {
+        "type": "string",
         "index": "not_analyzed",
-        "index_options" : "docs"
+        "omit_norms": true,
+        "index_options": "docs"
       },
-      "BaseURL" : {
-        "type" : "string",
+      "PromulgatedURL": {
+        "type": "string",
         "index": "not_analyzed",
-        "index_options" : "docs"
+        "index_options": "docs"
       },
-      "User" : {
-        "type" : "string",
-        "index" : "not_analyzed",
-        "omit_norms" : true,
-        "index_options" : "docs"
-      },
-      "Name" : {
-        "type" : "string",
-        "index" : "not_analyzed",
-        "omit_norms" : true,
-        "index_options" : "docs"
+      "BaseURL": {
+        "type": "string",
+        "index": "not_analyzed",
+        "index_options": "docs"
       },
-      "Revision" : {
-        "type" : "integer",
-        "index" : "not_analyzed"
+      "User": {
+        "type": "string",
+        "index": "not_analyzed",
+        "omit_norms": true,
+        "index_options": "docs"
       },
-      "Series" : {
-        "type" : "multi_field",
-        "fields" : {
-          "Series" : {
-            "type" : "string",
-            "index" : "not_analyzed",
-            "omit_norms" : true,
-            "index_options" : "docs"
+      "Name": {
+        "type": "multi_field",
+        "fields": {
+          "Name": {
+            "type": "string",
+            "index": "not_analyzed",
+            "omit_norms": true,
+            "index_options": "docs"
           },
-          "ngrams" : {
-            "type" : "string",
-            "analyzer" : "n3_20grams",
-            "include_in_all" : false
+          "ngrams": {
+            "type": "string",
+            "index_analyzer": "n3_20grams",
```

Comment:
This change from analyzer to index_analyzer, does it mean it was defaulting to some other analyzer and we never were using n3_20grams on this field?